infra/terraform/dev/deploy/vpc.tf

```diff
@@ -0,0 +1,112 @@
+# VPC
+
+resource "aws_vpc" "wikijump_vpc" {
+  cidr_block                       = var.vpc_cidr_block
+  enable_dns_hostnames             = true
+  enable_dns_support               = true
+  assign_generated_ipv6_cidr_block = true
+}
+
+# Subnets
+
+resource "aws_subnet" "elb_subnet" {
+  vpc_id                  = aws_vpc.wikijump_vpc.id
+  cidr_block              = var.elb_subnet
+  map_public_ip_on_launch = true
+  availability_zone       = format("%s%s", var.region, var.availability_zone)
```

Comment:
Curious, is there a difference between this and `"${var.region}${var.availability_zone}"`?